### ISSUE TITLE
Weird "0"s when setting inccommand option

### ISSUE THREAD

**Issue by rockyzhang24** [liuchengxu/eleline.vim#30]
After setting `set inccommand=split`, I see three weird "0"s when I run `:substitute`. I fully checked the source code but I couldn't figure it out.

I really like this lightweight statusline configuration. Thank you for your work 👍

![image](https://user-images.githubusercontent.com/11582667/91777991-2705a000-eba6-11ea-867d-d9499b7193c2.png)

UPDATE:
I found that if I didn't use the `def()` function which contains `printf()` in it, i.e., hardcode the statueline item using string concat, e.g., changing `let l:branch = s:def('ElelineGitBranch')` to `let l:branch = '%#ElelineGitBranch#' . ElelineGitBranch() . '%*'`, the issue would be solved.  But the git branch information on the statusline failed to refresh. 

UPDATE2:
I found I fixed it and it works well so far. I noticed that all the if conditions are written in a way of vim pipe in the same line. I changed the pipe way to the normal way, i.e., writing in multiple lines. For example: `if !exists('g:LanguageClient_loaded') | return '' | endif` is changed to:
```
if !exists('g:LanguageClient_loaded')
    return ''
endif
```
After changing all the pipe stuff, I found this issue disappeared. I don't know why, and I am not sure whether what I have done is correct.

Any ideas @liuchengxu ? Thank you very much.

**Comment by liuchengxu** [comment]
Theoretically that makes no differences, otherwise it's probably an issue of vim itself.

**Comment by rockyzhang24** [comment]
@liuchengxu Thanks. Yeah, that's confusing. To test it, last night I reverted the code back to the original, i.e., all if conditions are in the pipe way, and then I noticed that the bug come back. Next, I thought the first yellow "0" is caused by the ElelineGitBranch() function. Then I changed all the if conditions in this function into the multiple-line way, i.e., the normal way. The first yellow "0" was gone. The next two purple "0"s are caused by function ElelineLCN() and ElelineCoc(). I applied the same changes to these two functions, and the issue was solved. 

I have no idea and don't know whether there is some internal difference between the pipe and the multiple-line code. Annoying, hum....
